msalautomationapp/build.gradle

```diff
@@ -8,6 +8,10 @@ if (project.hasProperty("distMsalVersion")) {
 
 android {
 
+    packagingOptions {
+        pickFirst  'META-INF/common4j.kotlin_module'
+    }
```

Comment:
Why do we need this?

Reply:
cc/ @rpdome In case he has seen this earlier with Kotlin

Reply:
No, I haven't :/

Reply:
(I vaguely remember us running into META-INF a long while back pre-kotlin, but i can't exactly remember what happened)

Reply:
Yeah I also saw this happen for native auth project